docs/topics/auth/default.txt

```diff
@@ -256,10 +256,11 @@ a model's ``Meta`` class, you can also create permissions directly. For
 example, you can create the ``can_publish`` permission for a ``BlogPost`` model
 in ``myapp``::
 
-    from myapp.models import BlogPost
     from django.contrib.auth.models import Permission
     from django.contrib.contenttypes.models import ContentType
 
+    from .models import BlogPost
```

Comment:
Leave alone. This looks like a shell example.

Reply:
ok